docs/navigation.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navbars</title>
+    <link rel="stylesheet" href="../components/navbar/navbar.css">
+    <meta name="theme-color" content="#2C2929">
+
+</head>
+
+<body>
+
+    
+    <nav class="nav z-index-max">
+        <div class="flex flex-row nav-logo flex-between flex-align-item-center ">
+            <div class="sidebar-logo flex flex-align-item-center">
+                <span class="material-icons icon-nav-menu" id="menu">
+                    menu
+                </span>
+                <span class="logo-text">Framer-Ui</span>
+            </div>
+            <div class="flex flex-wrap gap">
+                <a href="https://github.com/mehulsatardekar"  target="_blank" rel="noreferrer">
+                    <box-icon name='github' type='logo' color='#eeeded' ></box-icon>  
+                </a>  
+                <a href="https://www.youtube.com/channel/UCOPdSGIn44Gh4QXZH_6U8gQ"  target="_blank" rel="noreferrer">
+                    <box-icon name='youtube' type='logo' color='#eeeded' ></box-icon>  
+                </a> 
+            </div>
+        </div>
+        <aside class="sidebar-menu  flex flex-column flex-between"  id="sidebar-menu">
+            <div class="flex flex-column flex-between mt-1">
+                <div class="flex flex-between flex-align-item-center ">
+                    <span class="font-mid-bold ml-1 logo-text">Framer-UI</span>
```

Comment:
You have used utility classes extensively which is great but you have named these classes a little bit understandable.  Navbar is looking great. Codes are clean but somewhere indentation is an issue. Other than that everything is great. Great work.